requirements.txt

```diff
@@ -4,3 +4,4 @@ PyPDF2~=3.0.1
 matplotlib~=3.10.0
 pandas~=2.2.3
 beautifulsoup4~=4.12.3
+atproto
```

Comment:
Don't we need to specify the version here?

Reply:
The package `atproto` is not in a major version yet and regularly being updated and patched. Since we plan for long term usage of it, it's better to get the latest version when possible:
- https://github.com/MarshalX/atproto/releases